File: 1878-CheckIfArrayIsSortedAndRotated/1878-CheckIfArrayIsSortedAndRotated.cpp
// Last updated: 8/13/2026, 10:12:10 PM
class Solution {
public:
    bool check(vector<int>& nums) {
        int count=0;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]>nums[(i+1)%nums.size()])
            count++;
        }
        return count<=1;
        
    }
};

File: 1878-CheckIfArrayIsSortedAndRotated/1878-CheckIfArrayIsSortedAndRotated.java
// Last updated: 8/13/2026, 10:12:03 PM
class Solution {
    public boolean check(int[] nums) {
        int count=0;
        int n=nums.length;
        for(int i=0;i<n;i++)
        {
            if(nums[i]>nums[(i+1)%n])
            count++;
        }
        return count<=1;
    }
}